tests/test_target.py

```diff
@@ -7,8 +7,16 @@ def test_constructor(self):
         self.assertIsNotNone(target)
         self.assertGreater(target.points, 0)
 
+    def test_targets_are_random(self):
+        target1 = Target()
+        target2 = Target()
+        #assert that the (x, y) coordinates are not the same
+        self.assertNotEqual((target1.rect.x, target1.rect.y), (target2.rect.x, target2.rect.y))
+
```

Comment:
This looks like you're testing that 2 targets are not created at the same location.  I do like this test because it shows they're not always created in the same place, but perhaps we could change the name.  The test_random_creation test seems similar to what your test name is implying.